policy/modules/kernel/filesystem.if

```diff
@@ -7014,8 +7014,10 @@ interface(`fs_rw_onload_sockets',`
 		type onload_fs_t;
 	')
 
+	allow $1 onload_fs_t:file {ioctl read write getattr};
```

Comment:
Could you please change line 7017 to:
rw_files_pattern($1, onload_fs_t, onload_fs_t)

Reply:
Thanks for the quick response.  I've made the change in 10a61df.  I'll make an equivalent change in https://github.com/containers/container-selinux/pull/66.